frontend/src/app/specification-overview/specification-overview.component.ts

```diff
@@ -38,6 +43,38 @@ export class SpecificationOverviewComponent implements OnInit {
     }
   }
 
+  public async downloadSpecification(specification: Specification, fileType: string) {
+    const latestVersion = specification.versions[0];
+    if (latestVersion !== undefined) {
+      this.downloadVersion[fileType](specification, latestVersion);
+    }
+  }
+
+  private createDownloadFileName(specification: Specification, version: Version) {
+    // Make it filename safe by replacing any non-alphanumeric character with an underscore
+    return (specification.title + '_v' + version.version).replace(/[^a-z0-9\.]/gi, '_');
```

Comment:
I'm not sure if I'd do this - they can change the file name; it is just a download after all. Or do you see a special case where this is important?

Reply:
Just thought it would be helpful, if you're downloading many at once.

Reply:
Now I see what you mean :) The first part where the version is added is indeed useful. I was talking about the regexp.

Reply:
Oh 😄 That's because I don't like spaces in my filenames...